### PR TITLE
Document delay injection in smartstack.yaml

### PR DESCRIPTION
**Pull request by yannlandry**
This adds documentation for the `fixed_delay` field in `smartstack.yaml`.

Please see the (internal, see branch name) related ticket for more information.

`make docs` passes with this as result:
screen shot 2019-01-04 at 11 02 12 am

### CODE REVIEW

File: docs/source/yelpsoa_configs.rst [hunk 1]
```diff
@@ -766,6 +766,36 @@ Routing and Reliability
...
+          ecosystem:stagec:
+            duration_ms: 3000
+            percent: 20
+          superregion:uswest1-prod:
+            duration_ms: 1000
+            percent: 40
+
+   - ``duration_ms``: The duration of the delay in milliseconds.
+   - ``percent``: Percentage of requests (0-100) to be randomly affected by this
+     delay. Note that due to Envoy's current definition of _percentage_ as an
```
Comment (bplotnick) [inline docs/source/yelpsoa_configs.rst]: I wonder why this isn't italicized?

**Comment by bplotnick** [comment]
lg2m! I filed #2096 since I think it should be more obvious in the documentation that a feature like this only works for Envoy (and is a noop elsewhere)

File: docs/source/yelpsoa_configs.rst (hunk 1 above)
Comment (yannlandry) [reply docs/source/yelpsoa_configs.rst]: Other italicized text uses `*text*` and no underscores. Maybe it's not supported. I fixed it now.

**Comment by solarkennedy** [comment]
In the long run, I would be OK if there was another place where these docs lived, and adjusted the paasta docs to point to them.

**Pull request merged**